### ISSUE TITLE
error when patching patching file src/LiberationMono-Regular.sfd

### ISSUE THREAD

**Issue by fiercebrake** [Frogging-Family/wine-tkg-git#1177]
Hey guys!

How would I go about fixing this particular problem ?

`Cloning into 'liberation-fonts'...
remote: Enumerating objects: 2094, done.
remote: Counting objects: 100% (26/26), done.
remote: Compressing objects: 100% (19/19), done.
remote: Total 2094 (delta 11), reused 13 (delta 7), pack-reused 2068
Receiving objects: 100% (2094/2094), 53.80 MiB | 21.83 MiB/s, done.
Resolving deltas: 100% (1431/1431), done.
HEAD is now at 9510ebd Merge pull request #1 from vishalvijayraghavan/master
patching file src/LiberationMono-Regular.sfd
fontforge -script scripts/fontexport.pe -ttf src/LiberationMono-Regular.sfd src/LiberationMono-Bold.sfd src/LiberationMono-Italic.sfd src/LiberationMono-BoldItalic.sfd src/LiberationSans-Regular.sfd src/LiberationSans-Bold.sfd src/LiberationSans-Italic.sfd src/LiberationSans-BoldItalic.sfd src/LiberationSerif-Regular.sfd src/LiberationSerif-Bold.sfd src/LiberationSerif-Italic.sfd src/LiberationSerif-BoldItalic.sfd
make: fontforge: No such file or directory
make: *** [Makefile:58: ttf-dir] Error 127
[proton-tkg] #> fontforge -script scripts/fontexport.pe
`

**Comment by Tk-Glitch** [comment]
Install the fontforge package.

**Comment by fiercebrake** [comment]
@Tk-Glitch 

Thanks for the tip!

i did install fontforge, but then I needed to install python-fonttools and rust, which seemed odd as I have never had to manually install all those packages. 

so what I did was to delete the folder where I was building everything, update the 'proton-tkg.cfg' file and rebuild the thing, this worked fine.

Again thank you for your lead! it was most definitely a problem on my side.

hope you have a good one!
